### PR TITLE
P1-snackbar-modal-branch

### PR DESCRIPTION
**Pull request by purohitdheeraj**
This Branch Contains 
- Snackbar Component
- Modal Component

### CODE REVIEW

File: library/components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    text-align: center;
+    flex-wrap: wrap;
+    gap: 2rem;
+}
+
+.auth-buttons .btn{
+    max-width: 60rem;
+}
+
+
```
Comment (zoyatanweer) [inline library/components/modal/modal.css]: you can remove extra spacing here

File: library/components/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+.modal-login{
+    display: none;
+}
+
+.modal-signup{
+    display: none;
+}
+.modal-wrapper{
+    position: relative;
+
```
Comment (zoyatanweer) [inline library/components/modal/modal.css]: remove extra line

**Review by zoyatanweer — APPROVED** [review]
very clean code
easy to understand
good to merge!

File: docs/documentation.html [hunk 3]
```diff
@@ -3481,6 +3661,135 @@ <h3>Usecase</h3>
...
+									class="snackbar__leading flex-row"
+								>
+									<div
+										class="snackbar__leading--text"
+									>
+										Your photo has been
+										archived.
+									</div>
+									<div
+										class="snackbar__leading--actions"
```
Comment (srishtigautam21) [inline docs/documentation.html:3725]: Why is your code so much towards the right side??

**Review by srishtigautam21 — APPROVED** [review]
The UI  looks great
The code looks good to me
Good to merge

File: docs/documentation.html (hunk 3 above)
Comment (purohitdheeraj) [reply docs/documentation.html:3725]: it's because of 4 indentations with Prettier

**Pull request merged**